src/4626/Factory4626Eth.sol

```diff
@@ -0,0 +1,93 @@
+pragma solidity 0.8.19;
+
+import {ERC20} from "solmate/tokens/ERC20.sol";
+
+import {MErc20} from "@protocol/MErc20.sol";
+import {MoonwellERC4626Eth} from "@protocol/4626/MoonwellERC4626Eth.sol";
+import {Comptroller as IMoontroller} from "@protocol/Comptroller.sol";
+
+contract Factory4626Eth {
+    /// ------------------------------------------------
+    /// ------------------------------------------------
+    /// ------------------ IMMUTABLES ------------------
+    /// ------------------------------------------------
+    /// ------------------------------------------------
+
+    /// @notice The Moonwell moontroller contract
+    IMoontroller public immutable moontroller;
+
+    /// @notice The WETH9 contract
+    address public immutable weth;
+
+    /// @notice The initial mint amount for a new vault
+    uint256 public constant INITIAL_MINT_AMOUNT = 0.01 ether;
```

Comment:
Does this need to be so high? It's ~$30 today but could be over $100 in a couple years. Why not 0.0001?